packages/components/addon/components/hds/app-footer/legal-links.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsAppFooterIndexComponent extends Component {
+  /**
+   * @param ariaLabel
```

Comment:
```suggestion
   * @param urlForSupport
```

Reply:
Oops, bad copy and pasting on my part. Thanks for catching all those.